app/presenters/subscription_presenter.rb

```diff
@@ -22,15 +41,19 @@ def available_filter_hash
     Hash[VacancyAlertFilters::AVAILABLE_FILTERS.collect { |v| [v, nil] }]
   end
 
-  def add_search_criteria_field(field, value)
+  # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Will Rubocop still complain if you make this into a [case expression](http://ruby-doc.com/docs/ProgrammingRuby/html/tut_expressions.html#S5)?

Reply:
Yup :)